SI/Sem.05/Pract.05/Task02.cpp

```diff
@@ -0,0 +1,64 @@
+﻿#include <iostream>
+using namespace std;
+
+
+/*
+ *Напишете програма, която подрежда елементите на редица от N числа, като първите елементи са четните, вторите - нечетните.
+Бонус: Сортирайте четните във възходящ ред,а нечетните в низходящ ред(отново четните са първи,а нечетните - втори).
+
+ */
+
+void sortArray(int (& arr)[], const int SIZE)
```

Comment:
no need of (& arr)[], just write int arr[], you are passing a pointer to the first element of the array, you are not passing the whole array